tools/project_gen/project_gen/project.py

```diff
@@ -111,7 +112,7 @@ def create_project(
                 / "db_creation"
                 / "empirical_db"
                 / "empirical_model_configs"
-                / "20p4.yaml"
+                / "21p10.yaml"
```

Comment:
Thoughts on making this a function argument instead of hardcoding it? @jasonmotha

Reply:
Happy with either tbh. Chances are we will only have one active version (especially in an automated sense) and the other reason to change it would be testing a new config.